### DISCUSSION TITLE
Trying to change colorscheme in kitty-diff yields strange behavior

### DISCUSSION THREAD

**Discussion by benjamin051000** [kovidgoyal/kitty#5653] [Q&A] [answered]
[Screencast from 11-09-2022 12:28:59 AM.webm](https://user-images.githubusercontent.com/34549440/200747164-5dac44f7-1495-4ed3-848e-bb58e619f79e.webm)

I am using the [sample diff.conf](https://sw.kovidgoyal.net/kitty/kittens/diff/#sample-diff-conf) file provided in the docs. 

I put this file in `~/.config/kitty/diff.conf`, and show that I have changed the `pygments_style` to `gruvbox-dark`. When I run diff, it opens in light mode (definitely not `gruvbox-dark`), and then changes to a washed out color palette a fraction of a second later. I've attached a video showing this behavior.

Is this a bug? Did I set something up incorrectly? 

Here's my Ctrl+Shift+F6, in case that's useful at all:
```
kitty 0.26.4 created by Kovid Goyal
Linux fedora 6.0.5-200.fc36.x86_64 #1 SMP PREEMPT_DYNAMIC Wed Oct 26 15:55:21 UTC 2022 x86_64
S
Kernel 6.0.5-200.fc36.x86_64 on an x86_64 (/dev/tty)

Running under: Wayland
Frozen: False
Paths:
  kitty: /usr/bin/kitty
  base dir: /usr/lib64/kitty
  extensions dir: /usr/lib64/kitty/kitty
  system shell: /usr/bin/zsh
Loaded config files:
  /home/benjamin/.config/kitty/kitty.conf

Config options different from defaults:
editor      nvim
font_family CaskaydiaCove Nerd Font
font_size   12.0

Important environment variables seen by the kitty process:
	PATH                                /home/benjamin/.cargo/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin
	LANG                                en_US.UTF-8
	EDITOR                              /usr/bin/nano
	SHELL                               /usr/bin/zsh
	DISPLAY                             :0
	WAYLAND_DISPLAY                     wayland-0
	USER                                benjamin
	XDG_MENU_PREFIX                     gnome-
	XDG_SESSION_DESKTOP                 gnome
	XDG_SESSION_TYPE                    wayland
	XDG_CURRENT_DESKTOP                 GNOME
	XDG_SESSION_CLASS                   user
	XDG_RUNTIME_DIR                     /run/user/1000
	XDG_DATA_DIRS                       /home/benjamin/.local/share/flatpak/exports/share:/var/lib/flatpak/exports/share:/usr/local/share/:/usr/share/
```

**Comment by kovidgoyal** [comment, marked as answer]
The sample is with *default* colors, which is a light theme. If you want
a dark theme you need to create one (or search in the issues there are
existing ones others have posted. pygments_style only sets the colors
for syntax highlighting.
